config/routes.rb

```diff
@@ -1,11 +1,9 @@
 Rails.application.routes.draw do
   devise_for :users
-  root 'landings#index'
-  resources :expenses
-  resources :categories
   resources :users
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'landings#index'
+  resources :categories do 
+    resources :expenses
+  end
```

Comment:
- [ ] Let's kindly use only the necessary routes in this project. As you can see in the screenshot below, some routes are not used.


![Screenshot from 2022-11-25 16-14-17](https://user-images.githubusercontent.com/63087491/204014321-f7d7dae3-730b-47ed-bf91-c8443b6cb0bb.png)